dawn/src/dawn/CMakeLists.txt

```
@@ -24,7 +24,8 @@ install(
 # GCC 8, still a common compiler, requires linking to a separate library for std::filesystem support.
 # Once that is no longer common we can remove this hack
 add_library(cpp_extra_libs INTERFACE)
-if (CMAKE_COMPILER_IS_GNUCC AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS 9.0)
+if ((CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX) AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS 9.0)
```

```suggestion
if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU" AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS 9.0)
```
I think it should be like that (check the c compiler as in the original version doesn't make sense...)

---

Yes, we remove the C compiler. This is just an artifact from before. Good catch!